### PR TITLE
feat: update to latest @octokit/rest

### PR DESCRIPTION
**Pull request by codebytere**
What it says on the tin. 

Updates to latest `@octokit/rest` so that we're using methods that match the latest docs.

/cc @BinaryMuse

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -53,7 +53,8 @@ const probotHandler = async (robot: Application) => {
 
         await context.github.checks.update(context.repo({
           name: existing.name,
-          check_run_id: `${existing.id}`,
+          owner: pr.base.repo.owner.login,
```
Comment (MarshallOfSound) [inline src/index.ts]: Why is this required?  `context.repo` should add this property I think

File: src/index.ts [hunk 2]
```diff
@@ -78,7 +79,8 @@ const probotHandler = async (robot: Application) => {
         label => label.name === `${labelPrefixes.target}${checkRun.name.replace(CHECK_PREFIX, '')}`,
       )) {
         context.github.checks.update(context.repo({
-          check_run_id: `${checkRun.id}`,
+          check_run_id: checkRun.id,
+          owner: pr.base.repo.owner.login,
```
Comment (MarshallOfSound) [inline src/index.ts]: Same here?

File: src/index.ts (hunk 1 above)
Comment (codebytere) [reply src/index.ts]: updated!
